qcodes/tests/dataset/test_dataset_basic.py

```diff
@@ -526,3 +526,23 @@ def test_get_description(some_paramspecs):
     loaded_ds = DataSet(run_id=1)
 
     assert loaded_ds.description == desc
+
```

Comment:
not sure that i understood your comment about snapshot. so you've made it both: non metadata AND not RUNS_TABLE_COLUMNS, right? and both should be fixed with upgrading the schema when snapshot is a necessary column of the runs table, right? if so, i agree. We should also do smth with metadata later so that arbitrary columns do not get created for metadata tags.